scripts/uberenv/specs.json

```diff
@@ -14,15 +14,13 @@
     "__comment__":"##############################################################################",
 
     "toss_3_x86_64_ib":
-    [ "clang@3.9.1",
-      "clang@4.0.0+devtools",
-      "gcc@4.9.3",
+    [ "clang@4.0.0+devtools",
+      "clang@6.0.0",
```

Comment:
Do we want to change the devtools to clang6? If so it could be done in another PR.

Reply:
It doesn't matter what compiler they are using.  Soon someone will rip them out of our axom package completely into their own spack package so its not tied to our compiler suites at all.